projects/plugins/jetpack/modules/likes/jetpack-likes-settings.php

```diff
@@ -711,41 +724,49 @@ function admin_settings_showbuttonon_init() {
 					$label = __( 'Front Page, Archive Pages, and Search Results', 'jetpack' );
 				} else {
 					$post_type_object = get_post_type_object( $show );
-					$label = $post_type_object->labels->name;
+					$label            = $post_type_object->labels->name;
 				}
 				?>
-				<?php if ( $br ) echo '<br />'; ?><label><input type="checkbox"<?php checked( in_array( $show, $global['show'] ) ); ?> name="show[]" value="<?php echo esc_attr( $show ); ?>" /> <?php echo esc_html( $label ); ?></label>
-				<?php	$br = true; endforeach; ?>
+				<?php
+				if ( $br ) {
+					echo '<br />';}
+				?>
+				<label><input type="checkbox"<?php checked( in_array( $show, $global['show'], true ) ); ?> name="show[]" value="<?php echo esc_attr( $show ); ?>" /> <?php echo esc_html( $label ); ?></label>
+				<?php
+				$br = true;
+endforeach;
```

Comment:
```suggestion
				endforeach;
```
Or it may be even clearer now to just use braces instead.